### PR TITLE
store/tikv: refine errors for undetermined result

### PR DESCRIPTION
**Pull request by hhkbp2**
Hi,

This PR refines errors for undetermined results. It returns key errors without wrapping when committing tasks.

PTAL @disksing @coocood @tiancaiamao

### CODE REVIEW

**Comment by hhkbp2** [comment]
A revisement for #3258

**Comment by shenli** [comment]
CI failed.

**Comment by disksing** [comment]
Can you add a test case for it?

**Comment by hhkbp2** [comment]
PTAL @disksing @coocood @tiancaiamao @AndreMouche

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -423,7 +438,7 @@ func (c *twoPhaseCommitter) doCommitSingleBatch(bo *Backoffer, batch batchKeys)
 	if regionErr != nil {
 		err = bo.Backoff(boRegionMiss, errors.New(regionErr.String()))
 		if err != nil {
-			return errors.Trace(err)
+			return wrapErr(isPrimary, err)
```
Comment (disksing) [inline store/tikv/2pc.go]: No need to make it undetermined here?

**Comment by disksing** [comment]
LGTM. PTAL @AndreMouche

**Comment by hhkbp2** [comment]
PTAL @coocood @tiancaiamao @AndreMouche

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
CI fail @hhkbp2

**Comment by hhkbp2** [comment]
PTAL @coocood @tiancaiamao

**Comment by hhkbp2** [comment]
thanks @coocood

**Pull request merged**